### ISSUE TITLE
Unexpected Error: "Minified React error #130"

### ISSUE THREAD

**Issue by tarcinil** [sensu/web#271]
## Expected Behavior
I see the issue with 2 points.

Visiting `/n/test/entities/test-agent-centos-7`, I should see the entities information.
Visiting `/n/test/events/test-agent-centos-7/keepalive`, I should see details about the event

## Current Behavior
Get "Something went Wrong"

## Steps to Reproduce (for bugs)
Entity:
1. Logged in via an Incognito
2. Browse to Entities
3. Click on Entity
4. Broke

Event:
1. Logged in via an Incognito
2. Browse to Event
3. Click on Entity's Event
4. Broke

## Context
This is running in a vagrant using private network.

**Browser Stack Trace:**

Entity:
```
Invariant Violation: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at ba (https://192.168.33.33/static/js/vendor_bcb1.js:152:438)
    at x (https://192.168.33.33/static/js/vendor_bcb1.js:152:541)
    at Ye (https://192.168.33.33/static/js/vendor_bcb1.js:152:42905)
    at https://192.168.33.33/static/js/vendor_bcb1.js:152:51475
    at S (https://192.168.33.33/static/js/vendor_bcb1.js:152:58950)
    at Qg (https://192.168.33.33/static/js/vendor_bcb1.js:152:63148)
    at Og (https://192.168.33.33/static/js/vendor_bcb1.js:152:62846)
    at Tg (https://192.168.33.33/static/js/vendor_bcb1.js:152:66679)
    at bi (https://192.168.33.33/static/js/vendor_bcb1.js:152:90700)
    at ci (https://192.168.33.33/static/js/vendor_bcb1.js:152:91084)
    at Di (https://192.168.33.33/static/js/vendor_bcb1.js:152:97974)
    at Yh (https://192.168.33.33/static/js/vendor_bcb1.js:152:97354)
    at Xh (https://192.168.33.33/static/js/vendor_bcb1.js:152:96375)
    at qf (https://192.168.33.33/static/js/vendor_bcb1.js:152:95246)
    at wg (https://192.168.33.33/static/js/vendor_bcb1.js:152:56094)
    at Object.next (https://192.168.33.33/static/js/lib_9292.js:13:38453)
```

Event:
```
Invariant Violation: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at ba (https://192.168.33.33/static/js/vendor_bcb1.js:152:438)
    at x (https://192.168.33.33/static/js/vendor_bcb1.js:152:541)
    at Ye (https://192.168.33.33/static/js/vendor_bcb1.js:152:42905)
    at https://192.168.33.33/static/js/vendor_bcb1.js:152:51475
    at S (https://192.168.33.33/static/js/vendor_bcb1.js:152:58950)
    at Qg (https://192.168.33.33/static/js/vendor_bcb1.js:152:63148)
    at Og (https://192.168.33.33/static/js/vendor_bcb1.js:152:62846)
    at Tg (https://192.168.33.33/static/js/vendor_bcb1.js:152:66679)
    at bi (https://192.168.33.33/static/js/vendor_bcb1.js:152:90700)
    at ci (https://192.168.33.33/static/js/vendor_bcb1.js:152:91084)
    at Di (https://192.168.33.33/static/js/vendor_bcb1.js:152:97974)
    at Yh (https://192.168.33.33/static/js/vendor_bcb1.js:152:97354)
    at Xh (https://192.168.33.33/static/js/vendor_bcb1.js:152:96375)
    at qf (https://192.168.33.33/static/js/vendor_bcb1.js:152:95246)
    at wg (https://192.168.33.33/static/js/vendor_bcb1.js:152:56094)
    at Object.next (https://192.168.33.33/static/js/lib_9292.js:13:38453)
```

Firefox Event
```
ba@https://192.168.33.33/static/js/vendor_bcb1.js:152:438
x@https://192.168.33.33/static/js/vendor_bcb1.js:152:541
Ye@https://192.168.33.33/static/js/vendor_bcb1.js:152:42906
Cf/<@https://192.168.33.33/static/js/vendor_bcb1.js:152:51475
S@https://192.168.33.33/static/js/vendor_bcb1.js:152:58950
Qg@https://192.168.33.33/static/js/vendor_bcb1.js:152:63148
Og@https://192.168.33.33/static/js/vendor_bcb1.js:152:62846
Tg@https://192.168.33.33/static/js/vendor_bcb1.js:152:66679
bi@https://192.168.33.33/static/js/vendor_bcb1.js:152:90702
ci@https://192.168.33.33/static/js/vendor_bcb1.js:152:91084
Di@https://192.168.33.33/static/js/vendor_bcb1.js:152:97974
Yh@https://192.168.33.33/static/js/vendor_bcb1.js:152:97356
Xh@https://192.168.33.33/static/js/vendor_bcb1.js:152:96375
qf@https://192.168.33.33/static/js/vendor_bcb1.js:152:95246
wg@https://192.168.33.33/static/js/vendor_bcb1.js:152:56096
next@https://192.168.33.33/static/js/lib_9292.js:13:38453
notifySubscription@https://192.168.33.33/static/js/vendor_bcb1.js:105:26227
onNotify@https://192.168.33.33/static/js/vendor_bcb1.js:105:26843
next@https://192.168.33.33/static/js/vendor_bcb1.js:105:27665
iterateObserversSafely/<@https://192.168.33.33/static/js/vendor_bcb1.js:90:195868
iterateObserversSafely@https://192.168.33.33/static/js/vendor_bcb1.js:90:195837
next@https://192.168.33.33/static/js/vendor_bcb1.js:90:195293
invoke@https://192.168.33.33/static/js/vendor_bcb1.js:90:209837
dMq0/</O</QueryManager.prototype.queryListenerForObserver/<@https://192.168.33.33/static/js/vendor_bcb1.js:90:211039
dMq0/</O</QueryManager.prototype.broadcastQueries/</<@https://192.168.33.33/static/js/vendor_bcb1.js:90:216828
dMq0/</O</QueryManager.prototype.broadcastQueries/<@https://192.168.33.33/static/js/vendor_bcb1.js:90:216805
dMq0/</O</QueryManager.prototype.broadcastQueries@https://192.168.33.33/static/js/vendor_bcb1.js:90:216756
dMq0/</O</QueryManager.prototype.fetchRequest/</b<@https://192.168.33.33/static/js/vendor_bcb1.js:90:218737
next@https://192.168.33.33/static/js/vendor_bcb1.js:105:28920
notifySubscription@https://192.168.33.33/static/js/vendor_bcb1.js:105:26227
onNotify@https://192.168.33.33/static/js/vendor_bcb1.js:105:26843
next@https://192.168.33.33/static/js/vendor_bcb1.js:105:27665
next/<@https://192.168.33.33/static/js/vendor_bcb1.js:90:217848
tryCatcher@https://192.168.33.33/static/js/vendor_bcb1.js:90:141139
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128795
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[16]</t.exports/PromiseArray.prototype._resolve@https://192.168.33.33/static/js/vendor_bcb1.js:90:134853
o/<[16]</t.exports/PromiseArray.prototype._promiseFulfilled@https://192.168.33.33/static/js/vendor_bcb1.js:90:135227
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129742
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[16]</t.exports/PromiseArray.prototype._resolve@https://192.168.33.33/static/js/vendor_bcb1.js:90:134853
o/<[16]</t.exports/PromiseArray.prototype._promiseFulfilled@https://192.168.33.33/static/js/vendor_bcb1.js:90:135227
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129742
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[16]</t.exports/PromiseArray.prototype._resolve@https://192.168.33.33/static/js/vendor_bcb1.js:90:134853
o/<[16]</t.exports/PromiseArray.prototype._promiseFulfilled@https://192.168.33.33/static/js/vendor_bcb1.js:90:135227
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129742
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[16]</t.exports/PromiseArray.prototype._resolve@https://192.168.33.33/static/js/vendor_bcb1.js:90:134853
o/<[16]</t.exports/PromiseArray.prototype._promiseFulfilled@https://192.168.33.33/static/js/vendor_bcb1.js:90:135227
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129742
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525
o/<[15]</n.exports/Promise.prototype._settlePromiseFromHandler@https://192.168.33.33/static/js/vendor_bcb1.js:90:128949
o/<[15]</n.exports/Promise.prototype._settlePromise@https://192.168.33.33/static/js/vendor_bcb1.js:90:129633
o/<[15]</n.exports/Promise.prototype._settlePromise0@https://192.168.33.33/static/js/vendor_bcb1.js:90:130370
o/<[15]</n.exports/Promise.prototype._settlePromises@https://192.168.33.33/static/js/vendor_bcb1.js:90:131733
o/<[15]</n.exports/Promise.prototype._fulfill@https://192.168.33.33/static/js/vendor_bcb1.js:90:130752
o/<[15]</n.exports/Promise.prototype._resolveCallback@https://192.168.33.33/static/js/vendor_bcb1.js:90:127525

```

## Your Environment

* Sensu version used (sensuctl, sensu-backend, and/or sensu-agent):
  * Agent: `sensu-agent version v5.18.1, build 0c45f08323ae43060e6c4a52fafc4da77bdf21d1, built 2020-03-11`
  * Backend: `sensu-backend version v5.18.1, build 0c45f08323ae43060e6c4a52fafc4da77bdf21d1, built 2020-03-11`
  * sensu/web revision: `ce8990b`
* Installation method (packages, binaries, docker etc.): I package my own RPM via omnibus
* Operating System and version (e.g. Ubuntu 14.04): 
  * backend/agent: CentOS 7
  * browser: OSX 10.14.6 via Chrome 80.0.3987.132 & Firefox 74.0

**Comment by georgespatton** [comment]
We're affected by this too.

**Comment by jamesdphillips** [comment]
@tarcinil is this happening when you try to view any of your agents? Or, does it seem to be limited to one or two of them? If possible would you be able to provide a dump of an entity where you're seeing this happen? `sensuctl entity info <my-entity> --format wrapped-json --namespace <namespace>`.

**Comment by georgespatton** [comment]
For us, it is any and all agents.  Here's an example entity:

 sensuctl entity info myhost-prod --format wrapped-json --namespace default

{
  "type": "Entity",
  "api_version": "core/v2",
  "metadata": {
    "name": "myhost-prod",
    "namespace": "default",
    "labels": {
      "environment": "production"
    }
  },
  "spec": {
    "deregister": true,
    "deregistration": {},
    "entity_class": "agent",
    "last_seen": 1584583221,
    "redact": [
      "password",
      "passwd",
      "pass",
      "api_key",
      "api_token",
      "access_key",
      "secret_key",
      "private_key",
      "secret"
    ],
    "sensu_agent_version": "5.18.1",
    "subscriptions": [
      "default",
      "mysub_production",
      "production",
      "entity:myhost-prod"
    ],
    "system": {
      "arch": "amd64",
      "cloud_provider": "",
      "hostname": "myhost-prod",
      "libc_type": "",
      "network": {
        "interfaces": [
          {
            "addresses": [
              "127.0.0.1/8",
              "::1/128"
            ],
            "name": "lo"
          },
          {
            "addresses": [
              "10.34.254.26/24",
              "fe80::20d:3aff:fe8b:a677/64"
            ],
            "mac": "mymac",
            "name": "eth0"
          }
        ]
      },
      "os": "linux",
      "platform": "ubuntu",
      "platform_family": "debian",
      "platform_version": "18.04",
      "vm_role": "",
      "vm_system": ""
    },
    "user": "agent"
  }
}

**Comment by tarcinil** [comment]
I don’t have my test instances anymore.  I’ll try to rebuild tomorrow. 

I was working via HTTPS bypassing security warning.

**Comment by adammcdonagh** [comment]
Hi,
I am also experiencing this issue. Same versions, built from source.

I can view the entities just fine, but get the issue when opening some event details.

e.g the keepalive event works just fine, but a custom check doesn't
https://sensuserver:3001/n/default/events/server1/check-processes is bad
https://sensuserver:3001/n/default/events/server1/keepalive is good

Here's what the dump of server1 shows:

```
{
  "type": "Entity",
  "api_version": "core/v2",
  "metadata": {
    "name": "server1",
    "namespace": "default"
  },
  "spec": {
    "deregister": false,
    "deregistration": {},
    "entity_class": "agent",
    "last_seen": 1585074004,
    "redact": [
      "password",
      "passwd",
      "pass",
      "api_key",
      "api_token",
      "access_key",
      "secret_key",
      "private_key",
      "secret"
    ],
    "sensu_agent_version": "",
    "subscriptions": [
      "unix-hosts",
      "rhel-hosts",
      "consul-server",
      "entity:server1"
    ],
    "system": {
      "arch": "amd64",
      "hostname": "server1",
      "network": {
        "interfaces": [
          {
            "addresses": [
              "127.0.0.1/8"
            ],
            "name": "lo"
          },
          {
            "addresses": [
              "REDACTED"
            ],
            "mac": "REDACTED",
            "name": "eth0"
          },
          {
            "addresses": [
              "REDACTED"
            ],
            "mac": "REDACTED",
            "name": "eth1"
          }
        ]
      },
      "os": "linux",
      "platform": "redhat",
      "platform_family": "rhel",
      "platform_version": "7.7"
    },
    "user": "agent"
  }
}
```

The agent itself has not yet been upgraded, so the agent version is showing blank. It's still 5.13.2

**Comment by jamesdphillips** [comment]
Thanks to everyone who was able to provide examples! 

I was able to replicate the issues on the production version and can confirm that the is issue has indirectly been fixed by #266. Fix should be available in `v5.19` which should ship on Monday.

Sorry for the inconvenience.

**Comment by toozej** [comment]
@jamesdphillips did the fix for this issue not make it into v5.19? 

I just upgraded to 5.19 this morning (from 5.18.1, compiled from source both times if that matters) and am still seeing this issue. Also the web revision didn't change `ce8990b`

**Comment by georgespatton** [comment]
Same here.

**Comment by jgrasser** [comment]
Yes, the web revision did not change.
https://github.com/sensu/sensu-go/blob/v5.19.0/dashboard/.ref

I had to dig a bit into the documentation - it was not easy to find - but it looks like it is possible to update this independently. See documentation here https://github.com/sensu/sensu-go/tree/v5.19.0/dashboard 

cc @toozej @georgespatton

**Comment by adammcdonagh** [comment]
I've just rebuilt using the 5.19.1 release.. Despite the changelog saying there's something fixed in web, the web version hasn't changed still...  😕

**Comment by jamesdphillips** [comment]
@toozej you are correct. Unfortunately, the fix didn't make it into the _open-source_ distribution of 5.19.X. The fix _has_ been pulled into the master branch of sensu-go (https://github.com/sensu/sensu-go/pull/3665), and if you build from master it should be working. If you are still encountering a crash afterward it's likely there is another unrelated issue.

Sorry about all the frustration this has caused.

**Comment by toozej** [comment]
@jamesdphillips any chance we could get a 5.19.2 release branch with the updated web assets from master in it? 

We build from the latest release branch for safety and reproducibility

**Comment by nikkixdev** [comment]
The team expects to have this shipped for 5.20.0 which we hope to have out soon!

**Comment by jamesdphillips** [comment]
The fix for this issue should appear in the v5.19.3 released earlier today. Thank you for your patience in resolving this issue.

**Comment by toozej** [comment]
I just confirmed on my cluster that upgrading to V5.19.3 updated the web UI and made the errors from this issue go away. Thanks so much @jamesdphillips @nikkixdev
